stable/satisfy/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: satisfy
-version: 0.1.1
+version: 0.1.2
 appVersion: "3.0.4"
```

Comment:
@anapsix please update the app version

Reply:
app version remained the same, changed were made to Nginx config of Docker image

Reply:
how do you suggest to proceed @cpanato, when docker image for application is updated, but application itself is not?